CollAction/CollAction.csproj

```diff
@@ -1,47 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-
   <PropertyGroup>
     <TargetFramework>netcoreapp1.1</TargetFramework>
     <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
     <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wp8+wpa81;</PackageTargetFallback>
-    <DisableImplicitFrameworkReference>true</DisableImplicitFrameworkReference>  
+    <DisableImplicitFrameworkReference>true</DisableImplicitFrameworkReference>
     <UserSecretsId>aspnet-CollAction-29a5d7b8-3530-4482-9ed6-572bd6c178fa</UserSecretsId>
     <DockerComposeProjectPath>..\docker-compose.dcproj</DockerComposeProjectPath>
-    <PreBuildEvent>npm install &amp;&amp; ^
```

Comment:
This means we don't get a rebuild frontend in visual studio. I'd prefer not to break other ways to test.

Reply:
Can PreBuildEvents only ben run for a given environment? Is there anyway we can do this just for production builds as we should not need this anymore in dev.

Reply:
@logophobia Hey Tim any thoughts on this one?

Reply:
Couple of options:
* Make it conditional so that you can disable with an evironment variable
* Make a separate csproj with that task disabled
* Disable it on build (not sure if that's possible with dotnet build)

https://stackoverflow.com/questions/1731904/can-you-prevent-msbuild-exe-from-running-build-events

Reply:
I like the environment variable idea. I can set it within the docker compose file. Will try and have a look at this one some time today.